lib/config/config.go

```diff
@@ -417,7 +419,7 @@ nextPendingDevice:
 		cfg.Options.AlwaysLocalNets = []string{}
 	}
 	if cfg.Options.UnackedNotificationIDs == nil {
-		cfg.Options.UnackedNotificationIDs = []string{}
+		cfg.Options.UnackedNotificationIDs = []string{"authenticationUserAndPassword"}
```

Comment:
This is not the right place to add this, as `clean` is called on every config change and we don't want to reintroduce it every time. What should happen as part of clean though is to check if user&password are set and if yes, remove `"authenticationUserAndPassword"` if present.

Reply:
Fixed as suggested